mandatory/4-tax.js

```diff
@@ -17,7 +20,12 @@ function calculateSalesTax() {}
   Remember that the prices must include the sales tax (hint: you already wrote a function for this!)
 */
 
-function addTaxAndFormatCurrency() {}
+function addTaxAndFormatCurrency(num) {
```

Comment:
This is amazing code. Making use of good naming conventions and ignoring unclear function names.